### PR TITLE
[CEDS-729] - added cancellation journey for integration.

### PR DESCRIPTION
**Pull request by raghu1978**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -51,7 +51,9 @@ class AppConfig @Inject() (override val runModeConfiguration: Configuration, val
   lazy val customsDeclarationsUrl = loadConfig("urls.customsDeclarations")
 
   lazy val customsDeclarationsEndpoint = baseUrl("customs-declarations")
-  lazy val submitImportDeclarationUri = getConfString("customs-declarations.submit-uri",
+  lazy val submitExportDeclarationUri = getConfString("customs-declarations.submit-uri",
+    throw new IllegalStateException("Missing configuration for Customs Declarations submission URI"))
+  lazy val submitCancellationUri = getConfString("customs-declarations.cancel-uri",
     throw new IllegalStateException("Missing configuration for Customs Declarations submission URI"))
```
Comment (PatrykRudnicki) [inline app/config/AppConfig.scala]: Change an exception to:
`"Missing configuration for Customs Declarations cancellation URI"`

File: app/controllers/CancelDeclarationController.scala [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                )
+              )
+            )
+        }
+      }
+    }
+  )
+}
+
+  private def createCancellationMetadata(form:CancelDeclarationForm):MetaData =
```
Comment (PatrykRudnicki) [inline app/controllers/CancelDeclarationController.scala]: We can stay with this here, but I think that we should create services to put methods inside services, not in Controller. We have a lot of code in controllers that we shouldn't have.

File: conf/application.conf [hunk 3]
```diff
@@ -78,7 +78,8 @@ microservice {
 
     customs-declarations {
       host = localhost
-      port = 6790
+     # port = 9820
```
Comment (PatrykRudnicki) [inline conf/application.conf]: Revert this file. Stay with correct port. No comments in `application.conf`

File: app/controllers/CancelDeclarationController.scala (hunk 2 above)
Comment (raghu1978) [reply app/controllers/CancelDeclarationController.scala]: this method will not exist after we have UI navigation built

File: conf/application.conf (hunk 3 above)
Comment (raghu1978) [reply conf/application.conf]: added comment for local testing to point to customs-declarations if needed

**Pull request merged**